### DISCUSSION TITLE
关于特性支持的问题

### DISCUSSION THREAD

**Discussion by dellbeat** [KonataDev/Konata.Core#165] [General]
在项目READ.ME内看到有部分特性是红色的
这些特性是**明确不会支持**（例如Mirai内明确不支持获取敏感信息）还是在**有可行性的前提**下会予以支持
希望能予以解答，谢谢！

**Comment by lepollo** [comment]
一切皆有可能

**Reply by ryuujo1573** [reply]
请问特性是如何实现的？如果感兴趣应该怎么参与贡献协议实现？

**Comment by dellbeat** [comment]
邮件已收到

**Comment by dellbeat** [comment]
邮件已收到

**Comment by TheSnowfield** [comment]
> 在项目READ.ME内看到有部分特性是红色的 这些特性是**明确不会支持**（例如Mirai内明确不支持获取敏感信息）还是在**有可行性的前提**下会予以支持 希望能予以解答，谢谢！

是正在plan中的feature, 以後可能會增加其支持。
